tests/core/math/test_vector2.h

```diff
@@ -372,6 +373,9 @@ TEST_CASE("[Vector2] Plane methods") {
 	CHECK_MESSAGE(
 			vector.project(vector_normal).is_equal_approx(Vector2(2.0292559899117276166, 0.60126103404791929382)),
 			"Vector2 projected on a normal should return expected value.");
+	CHECK_MESSAGE(
+			vector.plane_project(p_d, vector_normal).is_equal_approx(Vector2(117.339048445267494, 330.028130938608797)),
+			"Vector2 plane proeject should return expected value.");
```

Comment:
```suggestion
	CHECK_MESSAGE(
			vector_normal.plane_project(p_d, vector).is_equal_approx(Vector2(8.75944, 5.638667)),
			"Vector2 plane proeject should return expected value.");
```
The plane should be defined via a normal and a distance, the parameter of this method is actually the vector to project. The values above are with p_d set to 10 (validated visually).